java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttMessageListener.java

```diff
@@ -0,0 +1,48 @@
+package com.cumulocity.generic.mqtt.client;
+
+import com.cumulocity.generic.mqtt.client.model.GenericMqttMessage;
+import org.java_websocket.framing.CloseFrame;
+
+import java.net.URI;
+
+/**
```

Comment:
suggestion: I'd prefer something a bit stronger and more directed here. My experiences from working with doc teams that review Javadocs is that we should make it clear _who_ implements the interface. In this case maybe something like 'Application developers must implement this interface and pass and instance of it to {@link GenericMqttSubscriber#subscribe(GenericMqttMessageListener)} in order to handle events received by the WebSocket client from the server'. 
Maybe that's too long winded - please disagree or improve further